db/migrations/20230226112820_create_users_table.sql

```diff
@@ -0,0 +1,20 @@
+-- migrate:up
+CREATE TABLE users (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(100) NOT NULL,
+  email VARCHAR(100) NOT NULL,
+  password VARCHAR(100) NOT NULL,
+  phone_number VARCHAR(100) NOT NULL,
+  is_host BOOLEAN NOT NULL,
+  kakao_id VARCHAR(100) NULL,
+  point_id INT NOT NULL,
```

Comment:
User가 가지고 있는 최종 point 값이여야 될 것 같습니다.
```suggestion
  point INT NOT NULL,
```